src/app/views/provisioning/order.html

```diff
@@ -45,6 +45,29 @@
         </tr>
         </tbody>
       </table>
+
+      <table class="table table-striped" id="license-table" ng-show="vm.subscription.product_pricing.license_based">
+        <tr>
+          <td>
+            <label>{{ 'mnoe_admin_panel.dashboard.provisioning.order.licenses' | translate }}:
+              <input type="number" name="input" ng-model="vm.subscription.max_licenses" min="1" ng-required="vm.subscription.product_pricing.license_based">
+            </label>
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            <label>{{ 'mnoe_admin_panel.dashboard.provisioning.order.total' | translate }}:
+              <span ng-hide="vm.subscription.product_pricing.pricing_type == 'payg'" ng-repeat="price in vm.subscription.product_pricing.prices | filterBy: ['currency']: vm.orgCurrency" class="badge">
```

Comment:
Should free plans be taken into account here as well?

Reply:
sure